docs(go): Recommend SCM source context for Go (#17684)
getsentry/sentry-docs: docs/platforms/go/common/source-context/index.mdx

---
title: Source Context
sidebar_order: 90
sidebar_section: configuration
description: Learn how to show source lines in Go stack traces.
---

Source context shows the lines around a frame in a stack trace so you can see the code that failed without leaving Sentry.

For Go, source context is provided through [SCM Source Context](/integrations/source-code-mgmt/source-context/) with code mappings.

## Setup

To use source context with Go:

1. Set up a supported SCM integration.
2. Configure code mappings for your project.
3. Enable source context for the project.

For setup details, see [SCM Source Context](/integrations/source-code-mgmt/source-context/).

## Troubleshooting Missing Source Context

> If stack traces are missing source lines, check the following:

Reviewer: ```suggestion
If stack traces are missing source lines, make sure that:
```

---
title: Source Context
sidebar_order: 90
sidebar_section: configuration
description: Learn how to show source lines in Go stack traces.
---

Source context shows the lines around a frame in a stack trace so you can see the code that failed without leaving Sentry.

For Go, source context is provided through [SCM Source Context](/integrations/source-code-mgmt/source-context/) with code mappings.

> The Go SDK no longer reads local source files at runtime to populate source context. If you want source lines in stack traces, set up SCM Source Context instead of bundling source files with your deployed binary.

Reviewer 2: I think we can remove this paragraph, something for the SDK changelog instead.
